utils/styleguide/setup.js

```diff
@@ -24,11 +24,7 @@ global.DEFAULT_THEME = DEFAULT_THEME;
 global.PALETTE = PALETTE;
 global.ThemeProvider = ThemeProvider;
 global.Grid = Grid;
-global.Row = styled(Row).attrs({
-  alignItems: 'center'
-})`
-  /* stylelint-disable-line block-no-empty */
-`;
+global.Row = Row;
```

Comment:
Would it be better to allow the prop override of

```jsx
global.Row = styled(Row).attrs(props => ({
  alignItems: props.alignItems || 'center'	
}))`	
  /* stylelint-disable-line block-no-empty */	
`;
```

For most of examples we have centered styling. This would allow the prop overrides to affect it correctly.

Reply:
I went through all the examples. Actually, we do not have centered styling for most.